amr-wind/wind_energy/ABLReadERFFcn.H

```diff
@@ -0,0 +1,17 @@
+#ifndef ABLREADERFFCN_H
+#define ABLREADERFFCN_H
+#include <functional>
+
+class MultiBlockContainer;
+namespace amr_wind {
+    class InletData;
+}
+
+using ReadERFFcn = std::function <void (
```

Comment:
Rename to `ReadERFFunction` and all other mentions of `FCN` to function.

Reply:
okay, will do